ChopChop/Models/Recipes/Recipe.swift

```diff
@@ -2,14 +2,14 @@ import Foundation
 import GRDB
 
 /// Note there is no relationship between steps and ingredients after parsing stage
-class Recipe: FetchableRecord {
+class Recipe: FetchableRecord, ObservableObject {
```

Comment:
wait actually i thought observableobject is not used in the models? (only used between viemodels and views). does anyone know

Reply:
Yes, I'm not sure why this is needed.

Reply:
This is so that after updating a recipe via the form and then going back to the recipe view page, the page will be updated. As the form page has no reference to the view page, the way the sync them is to do it through the common model object shared between them.